src/streetview/component.js

```diff
@@ -225,6 +234,61 @@ class StreetviewController {
    * Called on initialization of the controller.
    */
   $onInit() {
+    // Initialization
+    if (this.featureStyle) {
+      this.feature_.setStyle(this.featureStyle);
+    }
+
+    /**
+     * @type {string}
+     */
+    const config = this.optionsName ? this.optionsName : 'ngeoStreetviewOptions';
+
+    /**
+     * @type {import('ngeo/options.js').ngeoStreetviewOptions}
+     */
+    this.options = this.injector_.has(config) ? this.injector_.get(config) : null;
+
+    /**
+     * @type {string}
+     */
+    this.viewer = this.options.viewer || 'mapillary';
```

Comment:
It would be better to define the property in the constructor (and not here, in the `onInit`) (type and `this.viewer = null`). Then the property at least always exists (which is better with angularjs in htmls.)
And here, just assign the value `this.viewer = this.options.viewer || 'mapillary';` without the typedoc (`@type {string}`)